packages/manager/apps/nasha-react/src/api/nasha-react/GET/iceberg/service.ts

```diff
@@ -0,0 +1,15 @@
+import { fetchIceberg } from '@ovh-ux/manager-core-api/src/iceberg';
```

Comment:
```suggestion
import { fetchIceberg } from '@ovh-ux/manager-core-api';
```

It should be exported from the package

Reply:
Done ✅